modules/internal/String.chpl

```diff
@@ -777,7 +811,9 @@ module String {
     */
     inline proc localize() : string {
       if _local || this.locale_id == chpl_nodeID {
-        return createStringWithBorrowedBuffer(this);
+        try! {
+          return createStringWithBorrowedBuffer(this);
```

Comment:
Can this call chpl_createString ?

Reply:
No need. Because there is no need for `try!` either. Factory functions with string arguments don't validate. I'll remove the `try!` block